lighthouse-core/test/audits/viewport-test.js

```diff
@@ -33,9 +35,17 @@ describe('Mobile-friendly: viewport audit', () => {
     }).rawValue, false);
   });
 
```

Comment:
Added some more tests for this.
Used JSON.stringify to print out the invalid property/values, sorry if there's a better way of doing this that I missed!